lib/trento_web/openapi/schema/host.ex

```diff
@@ -69,6 +69,23 @@ defmodule TrentoWeb.OpenApi.Schema.Host do
           description: "A list of the available SLES Subscriptions on a host",
           type: :array,
           items: SlesSubscription
+        },
+        deregistered_at: %Schema{
+          title: "DeregisteredAt",
+          description: "Timestamp of the last deregistration of the host",
+          type: :string,
+          nullable: true,
+          format: :"date-time"
+        },
+        heartbeat_timestamp: %Schema{
```

Comment:
Same consideration as above, the association and perhaps the entity field should be hearbeat, this is not a timestamp and is misleading